fastapi/params.py

```diff
@@ -1,5 +1,8 @@
 from enum import Enum
-from typing import Any, Callable, Dict, Optional, Sequence
+from typing import Any, Callable, Dict, Optional, Sequence, TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from fastapi.dependencies.models import DependencyLifespan
```

Comment:
I was getting circular imports, I'm not via what path. Since these are just type hints, this solves it, but I'm open to moving the aliases or another solution.